src/utils/connection.ts

```diff
@@ -1,6 +1,7 @@
+import { mixed } from "io-ts";
 import * as url from "url";
 
-export default (str: string) => {
+export const parseConnectionString = (str: string): mixed => {
```

Comment:
We might consider replacing this module with https://www.npmjs.com/package/pg-connection-string